### DISCUSSION TITLE
[Feature Request] 关于 Vue 3.2 新的 API defineCustomElement

### DISCUSSION THREAD

**Discussion by yingzhi0808** [youzan/vant#9312] [Ideas]
Vue 3.2 提供了一个新的 API `defineCustomElement` 可以直接将 Vue 组件编译为 Web Components，这样就可以用在非 Vue 的框架中，请问 Vant 会跟进吗？
相关链接：https://v3.vuejs.org/guide/web-components.html#using-custom-elements-in-vue

**Comment by chenjiahan** [comment]
后续会评估下喔，得实测看看效果如何

**Comment by zdxxxxxx** [comment]
这个有进展吗
